functions.photon.php

```diff
@@ -90,6 +90,22 @@ function jetpack_photon_url( $image_url, $args = array(), $scheme = null ) {
 		$args = rawurlencode_deep( $args );
 	}
 
+	/* Don't photon-ize WPCOM hosted images with only the following url args:
+	 *  `w`, `h`, `fit`, `crop`, `resize`
+	 * These args can just be added to the wpcom-version of the image, and save on latency.
+	 */
+	$is_wpcom_image_with_safe_args = false;
+	$allowed_wpcom_keys = array(
+		'w'      => true,
+		'h'      => true,
+		'fit'    => true,
+		'crop'   => true,
+		'resize' => true,
+	);
+	if ( wp_endswith( '.files.wordpress.com', $image_url_parts['host'] ) && array_diff_key( $args, $allowed_wpcom_keys ) ) {
```

Comment:
Is `wp_endswith()` a wpcom-only function? Looking on W.org only brings up [this old ticket](https://core.trac.wordpress.org/ticket/19347).